source/Octopus.Tentacle/Scripts/IScriptStateWorkspace.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Octopus.Tentacle.Scripts
+{
+    public interface IScriptStateWorkspace
```

Comment:
I wonder if this should be named `IStoreScriptState`.

The Workspace I think is just a place we happen to store the state.